### ISSUE TITLE
ERROR: Install failed. Got error "ApplicationVerificationFailed".

### ISSUE THREAD

**Issue by ghost** [iMokhles/IPAInstaller-Windows#4]
I keep trying to install this program on my iOS 6.1.6 iPod Touch 4th gen, and get application verification errors. Here is the log for install.
Click Extract now
Extracted 48 files, skipping 0 because they exist, with 0 errors. 
Extracting finished: Click Install now 
WARNING - Program will hang till the installation finish 
Due to the app size 
Don't worry the app will appear ASAP in your device 
"C:\Users\USER\AppData\Local\Temp/ipainstaller/ideviceinstaller.exe" -i "C:\Users\USER\AppData\Local\Temp/ipainstaller/ExtractedPath/Payload\xrec.app" 
Uploading xrec.app package contents... DONE.
Installing '(null)'
Install: CreatingStagingDirectory (5%)

Install: ExtractingPackage (15%)

Install: InspectingPackage (20%)

Install: TakingInstallLock (20%)

Install: PreflightingApplication (30%)

Install: VerifyingApplication (40%)
ERROR: Install failed. Got error "ApplicationVerificationFailed".

Installing - DONE

**Comment by ghost** [comment]
Yes i know i opened and closed an issue earlier, i did not know what was wrong, but now i do (I am very new to this program and was stupid enough to not look in the log.)

**Comment by hari49951** [comment]
@xAsrielDreemurr Hey I am getting this error. What to do in order to resolve the issue?
